samples/cpp/build_samples_msvc.bat

```diff
@@ -69,9 +65,9 @@ exit /b
 echo Build OpenVINO Runtime samples
 echo.
 echo Options:
-echo   -h                       Print the help message
-echo   -b SAMPLE_BUILD_DIR      Specify the sample build directory
-echo   -i SAMPLE_INSTALL_DIR    Specify the sample install directory
+echo   -h                        Print the help message
+echo   -b SAMPLES_BUILD_DIR      Specify the samples build directory
+echo   -i SAMPLES_INSTALL_DIR    Specify the samples install directory
```

Comment:
metavars diverge now with .sh: SAMPLES_BUILD_DIR vs SAMPLE_BUILD_DIR in .sh. Align them

Reply:
fixed